### PR TITLE
add route: can now add tasks on To Do List

### PR DESCRIPTION
**Pull request by mistakeets**
The front end can now see the tasks in the database table. I can add tasks on my To Do List, and they will appear in a list on the front end. index.pug and index.js updated.

### CODE REVIEW

**Comment by Moniarchy** [comment]
@teqnickels  @mistakeets I see that y'all have already merged this to master. Do you need a review on this still?

**Comment by mistakeets** [comment]
Yes please. We merged to master so we could continue working on another feature, however we would still like a review

**Comment by Moniarchy** [comment]
Cool. In the future, you should make sure that you get a review on the branch first before you merge--that's how git flow works on larger OSS projects. Master should always be in a stable state and the branches should be for features discrete enough that you could continue working on a different feature without worrying about the current one.

**Comment by teqnickels** [comment]
Thanks @Moniarchy, when you have a moment, can you show us how to do that. We're about to start a new branch now and work on the delete functionality. We'll submit a pull request for that branch and won't merge that one. From there... unsure what to do, if we need the changes made on that branch; could you show us? References are also great. Thanks

File: routes/index.js [hunk 1]
```diff
@@ -17,7 +23,7 @@ router.post('/createTask', (req,res)=> {
   });
 });
 
-// //GET ALL
+//GET ALL
```
Comment (jrobcodes) [inline routes/index.js:26]: Consider removing comments before submitting a PR/merging into master.

File: routes/index.js [hunk 2]
```diff
@@ -5,8 +5,14 @@ const Queries = require( '../database/queries.js' )
 const pgPromise = require('pg-promise')
 
 // /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get('/', function( req, res ) {
+  Tasks.getAll().then(Tasks => {
```
Comment (jrobcodes) [inline routes/index.js:9]: When I see identifiers that begin with a capital letter, I think it is a class name.  Best practice is to name types (classes) beginning with capital letters, and identifiers beginning with lower case capital letters.  Also, I think your catch block is associated with the incorrect block - it should be on the promise chain, not the call to `render`
```js
Tasks.getAll()
  .then( tasks => {
    response.render( 'index', { title: 'To Do List', tasks })
  })
  .catch( error => next( error ) )
```

**Pull request merged**